src/algorithms/post/election_post.md

```diff
@@ -63,34 +66,35 @@ The miner calls `GenerateCandidates` from proofs with their non-faulted (declare
 
 4. **(check Challenge Ticket(s) for winners)**
 Given a returned PartialTicket, miner checks it is a winning ticket. Specifically, they do the following:
-    - `ChallengeTicket = Finalize(PartialTicket).  = H(ChallengeTicket) / 2^len(H)`
+
+    - `ChallengeTicket = Finalize(PartialTicket) = H(ChallengeTicket) / 2^len(H)`
     - Check that `ChallengeTicket < Target`
+        - `Target = activePowerInSector/networkPower * 1/sectorsSampled * EC.ExpectedLeaders`.
+        - Put another way check `challengeTicket_num * networkPower * sectorsSampled_denom < activePowerInSector * sectorsSampled_num * EC.ExpectedLeaders * challengeTicket_denom`
```

Comment:
nit: should sectorsSampled_num here be sectorsSampled_denom and vice versa?

Reply:
I don't believe so. E.g. we might get 1/25 as sectors sampled or num = 1, denom = 25. We want to increase likelihood by the inverse, as in multiply left side by denom. 

So It's not nit but it is correct i think

Reply:
I don't think that's right -- multiplying the left hand side by 25 would make it 25x _less_ likely for any individual sample to hit the target.

Reply:
yes indeed. thanks